### ISSUE TITLE
questions when using the `vcf2smc`

### ISSUE THREAD

**Issue by guzhongru** [popgenmethods/smcpp#18]
Dear Jonathan
I am a little confused about the "distinguished lineages" when using the `smc++ vcf2smc`. The command is `smc++ vcf2smc my.data.vcf.gz out/chr1.smc.gz chr1 Pop1:S1,S2,S3`. The results showed that the distinguished lineages is S1 and undistinguished lineages is S2 and S3. So, does this mean that the first individual will set as distinguished lineages by default ? If I change the individual order, does it have an influence on the results ?

Cheers
Zhongru

**Comment by terhorst** [comment]
Yes, that is the default. If you changed the order, the distinguished lineages would change. In response to some other user requests I'm working on an update to vcf2smc which will make it more clear which lineages go where. I should have it done in a few days.

**Comment by guzhongru** [comment]
Ok, thanks for your answer. So, does it have an influence on the results if I set different individual as distinguished lineages ?

Cheers
Zhongru

**Comment by terhorst** [comment]
Yes, the results will be different (though probably similar) if you change the distinguished lineages.

**Comment by guzhongru** [comment]
So,,, if different, which one should be set as distinguished lineages ... Because the similarity among my studied populations is quite small.

**Comment by terhorst** [comment]
From a statistical point of view it does not matter. Select a sample at random.

**Comment by guzhongru** [comment]
Hi Jonathan
When I set different individual as distinguished lineages, I found the results have a significant difference. Specifically, I have 20 individuals belong to 2 population (A and B). They are differentiated at 5 kyr before present. But when I change the setting of distinguished lineages within each populations, the difference within population is larger than among populations. So, I don't know how to explain this.
